provisioning-telephony/config.yml

```diff
@@ -0,0 +1,72 @@
+---
+title: Provisioning Telephony
+description: This tutorial implements BYOC Cloud solution with a cloud-based carrier 
+categories:
+- Trunk
+- Telephony
+steps:
+- title: Introduction
+  content: |
+    This tutorial walks through the steps on how to implement BYOC Cloud in Genesys Cloud using [Purecloud Platform API](https://developer.mypurecloud.com/api/rest/client-libraries/index.html). In order for BYOC Cloud to work with a third-party carrier, the carrier must:
+    
+    * Be SIP-compliant
+    * Support the UDP or TCP trunk transport protocol
+    * Have a publicly routable IP address or host name
+      * Carriers must fully support Record-Routing as required by RFC 3261 to ensure proper routing of sequential SIP requests.
+    
+    For more information about carrier requirements, check Genesys Resource Center article [Carrier requirements for BYOC Cloud](https://help.mypurecloud.com/articles/carrier-requirements-byoc-cloud/).
+    
+    To successfully run the code, a JSON file is needed to provide the necessary input. In this tutorial, the JSON file is titled input-template.json.
+
+- title: JSON Files and Variables
+  content: |
+    This file is used as substitute to input text box. This is where the user enters all the necessary details. Key value pair has its corresponding description. Please make sure to replace all the value of keys from description to actual data and fill all the required fields to proceed. 
+    
+    Please refer to [BYOC Cloud public SIP IP addresses](https://help.mypurecloud.com/articles/byoc-cloud-public-sip-ip-addresses/) for instance or Public SIP Addresses.  
+
+- title: Authentication
+  content: |
+    Genesys Cloud will verify user's credentials using Client Credential authentication. Once the validation is successful, it will proceed with the succeeding process.
+
+- title: Check BYOC Feature
+  content: |
+    Get the list of products available for the org used by calling [getAuthorizationProducts()](https://developer.mypurecloud.com/api/rest/client-libraries/javascript/AuthorizationApi.html#organizationproductentitylisting_getauthorizationproducts__) function of Authorization API and check if the org has BYOC capability. If the org is BYOC capable, it will proceed to create location function.  Please take note that only BYOC enabled orgs can proceed.
+
+- title: Create Location
+  content: |
+    Create a location in Genesys Cloud  by calling [postLocations()](https://developer.mypurecloud.com/api/rest/client-libraries/javascript/LocationsApi.html#postLocations) function of the locations API and save the returned location data in a global variable.
+
+- title: Create Site
+  content: |
+    To create a new site, two pieces of information is needed. One is the location information, which is saved in a global variable and second is a default edge group which is found in PureCloud Admin > Telephony > Edge Groups.
```

Comment:
The procedure in DCA is different than production at this moment. In both cases we should not need the customer to reference the Edge Group.

DCA Environment: 
During creation of the Site the user will have the option to select the media model. For BYOC applications the media model must be **Cloud**. The location will reference the location they created in the previous step. 

PROD Environment:
During creation of the Site the user must select the radio option **Use these Sites** under the **Phone Edge Assignments** section. The managed site **PureCloud Voice** should be automatically filled. The location will reference the location they created in the previous step.

Reply:
👍 Just to make sure we're all on the same page, we should only convey instructions for production in this tutorial; it's customer-facing.

Reply:
should we reference/discuss the steps on PROD Environment? 
We tried focusing on the programming steps.

Reply:
Making mention of required configuration steps is a good idea, but this tutorial shouldn't focus on how to accomplish those tasks. Linking to some resource center articles on how to configure something will be helpful.

Reply:
we have link on about byoc cloud on the introduction. I think that covers the manual configuration.